packages/proto-rpc/src/ClientTransport.ts

```diff
@@ -0,0 +1,138 @@
+import {
+    ClientStreamingCall,
+    ServerStreamingCall,
+    DuplexStreamingCall,
+    Deferred,
+    RpcTransport,
+    MethodInfo,
+    RpcError,
+    RpcOptions,
+    RpcMetadata,
+    RpcStatus,
+    UnaryCall,
+    mergeRpcOptions
+} from '@protobuf-ts/runtime-rpc'
+import { v4 } from 'uuid'
+import { TODO } from './types'
+import { RpcMessage } from './proto/ProtoRpc'
+import EventEmitter = require('events')
+import { Logger } from './Logger'
+
+export enum Event {
+    RPC_REQUEST = 'streamr:dht-transport:request-new'
```

Comment:
Shorter value would suffice, e.g.
```suggestion
    RPC_REQUEST = 'rpcRequest'
```